docs/modules/data-structures/pages/updating-map-entries.adoc

```diff
@@ -16,7 +16,7 @@ Java::
 --
 [source,java]
 ----
-Map<Integer, String> countries = hzClient.getMap( "countries" );
+Map<Integer, String> countries = hzClient.getMap("countries" );
```

Comment:
I think that space was intentional. Wdyt?

Reply:
Yes, let's not touch the non-Go client bits in the doc.